code/lib/cli/src/sandbox-templates.ts

```diff
@@ -165,11 +165,8 @@ const baseTemplates = {
   },
   'vue2-vite/2.7-js': {
     name: 'Vue2 Vite (vue 2.7 JS)',
-    // TODO: convert this to an `npm create` script, use that instead.
-    // We don't really want to maintain weird custom scripts like this,
-    // preferring community bootstrap scripts / generators instead.
-    script:
-      'yarn create vite . --template vanilla && yarn add --dev @vitejs/plugin-vue2 vue-template-compiler vue@2 && echo "import vue2 from \'@vitejs/plugin-vue2\';\n\nexport default {\n\tplugins: [vue2()]\n};" > vite.config.js',
+    script: 'npx create-vue@2 {{beforeDir}} --default',
```

Comment:
Based on https://github.com/vuejs/create-vue, it looks like we could also use `npm create vue@3`, right?  Especially since SB7 does not support IE.

Reply:
I guess it's unclear to me based on that readme whether that's the version of the creator or the version of vue that will be installed.  It only mentions IE support as the difference between the two.

Reply:
@IanVS with create-vue@3 it does generate a Vue3 Vite project:
```
{
  "scripts": {
    "dev": "vite",
    "build": "vite build",
    "preview": "vite preview"
  },
  "dependencies": {
    "vue": "^3.2.45"
  },
  "devDependencies": {
    "@vitejs/plugin-vue": "^4.0.0",
    "vite": "^4.0.0"
  }
}
```

The docs states:
> Or, if you need to support IE11, you can create **a Vue 2 project** with